src/main/java/com/sourcegraph/webhook/registry/WebhookRegistry.java

```diff
@@ -101,7 +109,7 @@ public static void register(Webhook hook) throws WebhookException {
                 if (!add.contains(event.getEvent())) {
                     activeObjects.delete(event);
                 }
-                add.remove(event);
+                add.remove(event.getEvent());
```

Comment:
Ohhh, this was it?

Reply:
Yup, the reason we shot down an entire bitbucket server 👀